index.html

```diff
@@ -0,0 +1,1117 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <main class="main">
+        <nav class="nav-bar nav-bar-h">
+            <div class="tnw-logo tnw-logo-h">
+                <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="tnw-logo">
+            </div>
+            <div class="right-nav">
```

Comment:
[Optional] Let's use more semantic tags like` header, aside` instead of just div tags. They are very useful in SEO and debugging purposes also. Please reference [this](https://www.w3schools.com/html/html5_semantic_elements.asp) for more info.